src/couchbeam_view.erl

```diff
@@ -406,9 +410,9 @@ parse_view_options([{skip, Value}|Rest], #view_query_args{options=Opts}=Args) ->
 parse_view_options([{list, Value}|Rest], #view_query_args{options=Opts}=Args) ->
     Opts1 = [{list, Value}|Opts],
     parse_view_options(Rest, Args#view_query_args{options=Opts1});
-parse_view_options([{keys, Value}|Rest], Args) ->
```

Comment:
Yes, there may be a problem. I've figured out that it doesn't work for me probably because I'm using Couchbase's Sync Gateway which supports limited API. On pure couch it should work fine, I'm going to exclude this change from pull request.